apps/aeutils/test/data/epoch_testnet.yaml

```diff
@@ -10,3 +10,5 @@ chain:
 
 mining:
     autostart: true
+    # Public key of beneficiary account that will receive fees from mining on a node.
+    beneficiary: "encoded_beneficiary_pubkey_to_be_replaced"
```

Comment:
This config should contain the bare minimum to join the testnet. Thus, I think the `beneficiary` should not be added in.

Reply:
@dincho BTW I made an assumption that, in order to force users to set it, encodable `beneficiary` must be present in either user config or sys config to start a node. Otherwise `aec_conductor` will fail (this is our "main" application process).

Reply:
In case of mandatory `beneficiary` is makes sense.